src/libraries/Microsoft.Extensions.Hosting/src/HostAbortedException.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Runtime.Serialization;
+
+namespace Microsoft.Extensions.Hosting
+{
+    /// <summary>
+    /// The exception that is thrown upon <see cref="IHost"/> abortion.
+    /// </summary>
+    [Serializable]
+    public sealed class HostAbortedException : Exception
+    {
+        private HostAbortedException(SerializationInfo info, StreamingContext context) : base(info, context) { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HostAbortedException"/> class
+        /// with a system-supplied error message.
+        /// </summary>
+        public HostAbortedException() { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HostAbortedException"/> class
+        /// with a specified error message.
+        /// </summary>
+        /// <param name="message">
+        /// The error message that explains the reason for the exception.
+        /// </param>
+        /// <remarks>
+        /// The content of <paramref name="message"/> is intended to be understood by humans.
+        /// The caller of this constructor is required to ensure that this string has been localized for the
+        /// current system culture.
+        /// </remarks>
+        public HostAbortedException(string? message) : base(message) { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HostAbortedException"/> class
+        /// with a specified error message and a reference to the inner exception that
+        /// is the cause of this exception.
+        /// </summary>
+        /// <param name="message">
+        /// The error message that explains the reason for the exception.
+        /// </param>
+        /// <param name="innerException">
+        /// The exception that is the cause of the current exception.
+        /// If <paramref name="innerException"/> is not <see langword="null"/>, the current exception
```

Comment:
I assume you copied this description from somewhere. Can you point me to where it was copied from?

Looking through other occurrences of `<param name="innerException">` in the repo, I'm not finding it. I just see

[https://github.com/dotnet/runtime/blob/fbef29b711d94258ea872e15309366c369e36630/src/libraries/System.IO.Packaging/src/System/IO/Packaging/FileFormatException.cs#L41-L42](https://github.com/dotnet/runtime/blob/fbef29b711d94258ea872e15309366c369e36630/src/libraries/System.IO.Packaging/src/System/IO/Packaging/FileFormatException.cs#L41-L42)

Reply:
I am not quite sure anymore. I mean it was OperationCanceledException.

Reply:
I don't even know what this means on OperationCanceledException. Looking at the history of its docs, that line comes from over 5 years ago, and was added in https://github.com/dotnet/dotnet-api-docs/commit/ca7d94d93ac693ef3a5d234cbceaf445cdc9ed35, which is too large to view.

Can you change the innerException docs to look like FileFormatException above?